features/332-sso/sign-in-with-apple-prerequisite.md

```diff
@@ -0,0 +1,96 @@
+# How to set up Sign in with Apple as a developer
```

Comment:
I'm not sure if we should include these instructions in this repo. These instructions does not seems specific to Skygear (which can be easily found by searching).

Reply:
All contents are searchable, but scattered in different places. The skygear specific instruction is the UserConfiguration example.